Google.PowerShell/BigQuery/GcbqTable.cs

```diff
@@ -112,4 +112,174 @@ protected override void ProcessRecord()
             }
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Creates a new empty table in the specified project and dataset.
+    /// </para>
+    /// <para type="description">
+    /// Creates a new empty table in the specified dataset. A Table can be supplied by object 
+    /// via the pipeline or the -InputObject parameter, or it can be instantiated by value 
+    /// with the flags below. The Dataset ID can be specified by passing in a string to 
+    /// -DatasetId, or you can pass a Dataset or DatasetReference to the -Dataset parameter. 
+    /// If no Project is specified, the default project will be used. This cmdlet returns 
+    /// a Table object.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> New-GcbqTable “new_tab” -Dataset “my_data” -Description “Some nice data!” 
+    ///         -Expiration (60*60*24*30)</code>
+    ///   <para>This makes a new Table called "new_tab" with a lifetime of 30 days.</para>
+    ///   <code>PS C:\> Get-GcbqDataset "my_data" | New-GcbqTable “new_tab”</code>
+    ///   <para>This shows how the pipeline can be used to specify Dataset and Project.</para>
+    /// </example>
+    /// <para type="link" uri="(https://cloud.google.com/bigquery/docs/reference/rest/v2/tables)">
+    /// [BigQuery Tables]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "GcbqTable")]
+    public class NewGcbqTable : GcbqCmdlet
+    {
+        private class ParameterSetNames
+        {
+            public const string ByObject = "ByObject";
+            public const string ByValues = "ByValue";
+            public const string ByValueWithRef = "ByValueWithRef";
+        }
+
+        /// <summary>
+        /// <para type="description">
+        /// The Table object that will be sent to the server to be inserted.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipeline = true, ParameterSetName = ParameterSetNames.ByObject)]
+        public Table InputObject { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// The project to put the table in. If not set via PowerShell parameter processing, will
+        /// default to the Cloud SDK's DefaultProject property.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false, ParameterSetName = ParameterSetNames.ByValues)]
+        [ConfigPropertyName(CloudSdkSettings.CommonProperties.Project)]
+        public override string Project { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// The DatasetId that you would like to add to. This field takes strings.
+        /// To pass in a Dataset or DatasetId object for this field, use the ByValuesWithRef parameter set.
```

Comment:
I don't know what a DatasetId object is. Just say Dataset object. You don't need to talk about `DatasetReference` objects here.

Also, don't talk about the name of the parameter set. The name is not visible to the users. Instead, just mention the Dataset parameter.